HW6/OsypenkoIryna/task1

```diff
@@ -0,0 +1,8 @@
+even_num = [num for num in range(10) if num % 2 == 0]
+print (even_num)
+
+odd_num = [num for num in range(10) if num % 3 == 0]
```

Comment:
according to the conditions of the problem, it is necessary to adjust this condition to the following
odd_num = [num for num in range(10) if num % 3 == 0 and num % 2 == 1]